src/webgpu/api/operation/copyBetweenLinearDataAndTexture.spec.ts

```diff
@@ -771,3 +863,64 @@ g.test('copy_various_mip_levels')
       checkMethod,
     });
   });
+
+// Test that when copying a single row we can set any bytesPerRow value and when copying a single slice we can set rowsPerImage to 0.
+// Origin, offset, mipLevel and rowsPerImage values will be set to undefined when appropriate.
+g.test('copy_with_no_image_or_slice_padding_and_undefined_values')
+  .params(
+    params()
+      .combine(kMethodsToTest)
+      .combine([
+        {
+          bytesPerRow: 0,
+          rowsPerImage: 0,
+          copySize: { width: 3, height: 1, depth: 1 },
+          origin: { x: 0, y: 0, z: 0 },
+        }, // copying one row: bytesPerRow and rowsPerImage can be set to 0
+        {
+          bytesPerRow: 256,
+          rowsPerImage: 0,
+          copySize: { width: 100, height: 1, depth: 1 },
+          origin: { x: 0, y: 0, z: 0 },
+        }, // copying one row: bytesPerRow can be < bytesInACompleteRow = 400
+        {
+          bytesPerRow: 256,
+          rowsPerImage: 0,
+          copySize: { width: 3, height: 3, depth: 1 },
+          origin: { x: 0, y: 0, z: 0 },
+        }, // copying one slice: rowsPerImage = 0 will be set to undefined.
```

Comment:
Can we also test that rowsPerImage = 2 is allowed and works even if less than copySize.depth?